### PR TITLE
Issue #13345 enabling avoidnestedblockcheckexampletest

### PR DESCRIPTION
**Pull request by piyush-1234**
Issue: #13345

### CODE REVIEW

File: src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheckExamplesTest.java [hunk 1]
```diff
@@ -34,18 +34,19 @@ protected String getPackageLocation() {
...
     }
 
     @Test
     public void testExample2() throws Exception {
         final String[] expected = {
-
+            "19:5: " + getCheckMessage(MSG_KEY_BLOCK_NESTED),
         };
 
-        verifyWithInlineConfigParser(getPath("Example2.txt"), expected);
```
Comment (romani) [inline src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheckExamplesTest.java:49]: Please remove such files at all from git.
We should have only java files.
Please use java files in template xml file.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheckExamplesTest.java (hunk 1 above)
Comment (piyush-1234) [reply src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheckExamplesTest.java:49]: sure. @romani

**Comment by romani** [comment]
@piyush-1234 , CI pointing to problem.

**Comment by piyush-1234** [comment]
> @piyush-1234 , CI pointing to problem.

fixing it

**Pull request closed without merging**